mirrord/cli/src/config.rs

```diff
@@ -26,6 +26,14 @@ pub(super) enum Commands {
         .multiple(true)
         .args(&["target", "config-file"]),
 ))]
+#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, ValueEnum)]
+enum FsMode {
+    ReadWrite,
+    Read,
+    Local,
```

Comment:
What's the difference between Local and Disabled?

Reply:
Local means you can still provide overrides while disabled means nothing is done using mirrord in terms of fs.

Reply:
So local without added overrides is the same as disabled? Seems redundant

Reply:
There are also defaults that are made by us (for example, aws cli cache) that'd read remotely even if user didn't include anything.
Also, what do you suggest as an alternative?

Reply:
So Disabled == Local without the default overrides? Best option I can think of is to have a separate parameter for "default_overrides". Otherwise it's really unclear what the difference between the two is, plus "Disabled" feels more like the "default" option.

Reply:
Disabled means that any overrides don't work, i.e no file hooks..